gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -584,6 +587,36 @@ private static void setupGenericConjureProjects(
         });
     }
 
+    /**
+     * Locates projects either as child projects or as peer projects whose names match the patterns given by
+     * the GENERIC_GENERATOR_LANGUAGE_NAMES_PROPERTY property.
+     */
+    private static Map<String, Project> findGenericDerivedProjects(Project project) {
+        Map<String, Project> genericSubProjects = Collections.emptyMap();
+        String projectName = project.getName();
+
+        boolean useFlatProjectStructure = project.hasProperty(USE_FLAT_PROJECT_STRUCTURE_PROPERTY);
+        if (!useFlatProjectStructure) {
+            genericSubProjects = Maps.filterKeys(project.getChildProjects(), childProjectName -> {
```

Comment:
style nit: Can we return early to avoid having to define the variable? Can also get rid of the else to lower the nesting a bit.

Reply:
heh - that is definitely a style difference for me.  I prefer single point of return when there is a possibility of a hanging "else".  But when in Rome...